neo/rawio/neuralynxrawio.py

```diff
@@ -89,7 +90,7 @@ def _parse_header(self):
             if ext not in self.extensions:
                 continue
 
-            if (os.path.getsize(filename)<=HEADER_SIZE) and (ext == 'ncs'):
+            if (os.path.getsize(filename)<=HEADER_SIZE) and (ext in ['ncs']):
```

Comment:
Do you mean
```python
(ext in ['ncs', 'ntt']):
```

Reply:
At first, there were 
`ext in ['ncs', 'nse', 'ntt']`
but then we had a talk in #553, and then I just put 'ncs'.  
But this version of mine lived in my local verison. Now I just wanted to sync with the master. 

So afaik, there were 2 reasons:
1. Empty NCS are not important
2. Empty spikes (NSE, NTT) are somehow important. However, we didn't ever used this info.

Therefore, I just skipped empty ncs and put empty info to nse and ntt

Reply:
Oups.
I reply to quickly.
I forgot the last patch and mix nse and nse.
I am sorry, this is OK.